pootle/apps/pootle_app/management/commands/__init__.py

```diff
@@ -17,6 +17,22 @@
 from pootle_translationproject.models import TranslationProject
 
 
+class SkipChecksMixin(object):
+    def check(self, app_configs=None, tags=None, display_num_errors=False,
+              include_deployment_checks=False):
+        skip_tags = getattr(self, 'skip_system_check_tags', None)
+        if skip_tags is not None:
+            from django.core.checks.registry import registry
+            tags = registry.tags_available()
```

Comment:
Also not sure how avoiding to go through all elements is considered to be more important than readability in [such a small set](https://docs.djangoproject.com/en/1.8/ref/checks/#builtin-tags) in non performance-critical code, when `tags = registry.tags_available() - set(skip_tags)` reads natural and gets the job done.